frontend/app/mocks/db.ts

```diff
@@ -104,6 +110,21 @@ for (let i = 0; i < numberOfLetters; i++) {
   });
 }
 
+// seed letter type list
+db.letterType.create({
+  id: faker.string.alphanumeric(10),
```

Comment:
You don't need to set the `id` since it will use `faker.string.uuid` as default.

```
id: primaryKey(faker.string.uuid),
```

Reply:
Okay, thanks. Removed in commit 26a1fa9b74af0e32cb83dd016a8c94fc4adc4e2f.